tests/test_objective_funs.py

```diff
@@ -956,6 +958,88 @@ def test(eq, field, correct_value, rtol=1e-14, grid=None):
 
         test(eq, field, psi_from_field)
 
+    @pytest.mark.unit
+    def test_signed_plasma_vessel_distance(self):
+        """Test calculation of signed distance from plasma to vessel."""
+        R0 = 10.0
+        a_p = 1.0
+        a_s = 2.0
+        # default eq has R0=10, a=1
+        eq = Equilibrium(M=3, N=2)
```

Comment:
Elsewhere in our tests we do things based off the default geometry, and it has not changed since the code's inception. I would rather keep this as it is.